slate/source/localizable/index.html.md

```diff
@@ -123,7 +123,7 @@ Examples of what the metadata includes are:
 
 ```shell
 # Exchange the client credentials (client ID & secret) for an access token
-curl --user {clientId}:{clientSecret}  -X POST -d 'grant_type=client_credentials' -d 'scope={scope}' https://ala-test.auth.ap-southeast-2.amazoncognito.com/oauth2/token
+curl --user {clientId}:{clientSecret}  -X POST -d 'grant_type=client_credentials' -d 'scope={scope}' https://auth.ala.org.au/cas/oidc/oidcAccessToken
```

Comment:
Does this have to updated for production everytime there's a merge?

Reply:
This is a old example which was added about a year ago. I've just updated it so it actually points to a current valid ala auth system. I will look at moving the auth urls to configs for the next update.

Reply:
Implemented here: https://github.com/AtlasOfLivingAustralia/ala-api-portal/commit/7251eba09a8c27e319e658e6682b95e1a83a43fa